packages/block-library/src/image/image-editor.js

```diff
@@ -259,6 +265,29 @@ export default function ImageEditor( {
 
 	return (
 		<>
+			{ ! inProgress && (
+				<div
+					className="richimage__zoom-control"
```

Comment:
Are we keeping these "richimage" classes? They don't mean much.

Reply:
Happy to clean them up. Any preference, just "image__"?

Reply:
I'd go with `wp-block-image__` since the block class is be `wp-block-image`.